lib/decorator.rb

```diff
@@ -0,0 +1,12 @@
+require './nameable'
+
+class Decorator < Nameable
```

Comment:
[OPTIONAL] It would be best if you added a getter and setter for the nameable variable using the `attr_accessor`.

```
class Decorator < Nameable
    attr_accessor :nameable
    ....
```

Reply:
Hi @KDlamini this is a great suggestion. I have added a getter and setter for the nameable variable using the ```attr_accessor```